tikv/tikv
# fuzz: Bump afl version

### What is changed and how it works?

Issue Number: Close #15847

What's Changed:

Bump `afl` version to include the fix of the original issue


```commit-message
    bump afl version

    Bump afl to a new version that can successfully builds on MacOS.

    close tikv/tikv#15847
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`: NA
- Need to cherry-pick to the release branch: No

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Run `cargo run --package fuzz -- run afl fuzz_codec_bytes` to make sure fuzz testing works with afl

Side effects

None

### Release note 

```release-note
None
```
